grammar.js

```diff
@@ -864,6 +911,31 @@ module.exports = grammar({
       optional(seq(',', $.output_item_list))
     ),
 
+    open_statement: $ => seq(
+      caseInsensitive('open'),
+      '(',
+      choice(
```

Comment:
I think we can use the `$.argument_list` expression here instead of hard coding out the argument options. It's a bit more abstract than defining all the options but also makes it less error prone to edge-cases in the allowable syntax which Fortran is full of